config.js

```diff
@@ -1,2 +1,2 @@
 export const PORT = process.env.PORT || 3000;
-export const MONGO_URL = process.env.MONGO_URL;
\ No newline at end of file
+export const MONGO_URL = "mongodb+srv://z:RG99XAwjOJsDJijG@agile.ctke1it.mongodb.net/agile?retryWrites=true&w=majority";
```

Comment:
bruh this is our database credential, why did u push this
once it is on Github someone will misuse it